python_modules/dagster/dagster/_core/definitions/decorators/op_decorator.py

```diff
@@ -417,9 +417,9 @@ def resolve_checked_solid_fn_inputs(
         undeclared_inputs_printed = ", '".join(undeclared_inputs)
         raise DagsterInvalidDefinitionError(
             f"{decorator_name} '{fn_name}' decorated function does not have parameter(s) "
-            f"'{undeclared_inputs_printed}', which are in provided input_defs. {decorator_name} "
-            "decorated functions should only have keyword arguments that match input names and, if "
-            "system information is required, a first positional parameter named 'context'."
+            f"'{undeclared_inputs_printed}', which are in provided ins. {decorator_name}-"
+            "decorated functions should have a keyword argument for each In name, except for Ins "
```

Comment:
sidenote but it's not true that they should have a keyword argument for each `In` right? It could be a non-kwarg too